From 9b08713a380c9bfa538b0afa2a94884740d3860a Mon Sep 17 00:00:00 2001
From: Robin Gong <b38343@freescale.com>
Date: Fri, 3 Apr 2015 19:26:56 +0800
Subject: [PATCH] MLK-10588: ARM: dts: imx7d: enable wdog1 by default

Enable wdog1 as default watchdog device since on all boards we use wdog1.

Signed-off-by: Robin Gong <b38343@freescale.com>
---
 arch/arm/boot/dts/imx7d.dtsi | 1 -
 1 file changed, 1 deletion(-)

diff --git a/arch/arm/boot/dts/imx7d.dtsi b/arch/arm/boot/dts/imx7d.dtsi
index f1901f9..0e3fa9c 100644
--- a/arch/arm/boot/dts/imx7d.dtsi
+++ b/arch/arm/boot/dts/imx7d.dtsi
@@ -443,7 +443,6 @@
 				reg = <0x30280000 0x10000>;
 				interrupts = <GIC_SPI 78 IRQ_TYPE_LEVEL_HIGH>;
 				clocks = <&clks IMX7D_WDOG1_ROOT_CLK>;
-				status = "disabled";
 			};
 
 			wdog2: wdog@30290000 {
-- 
1.8.0

